include/Benchmark/Benchmark.h

```diff
@@ -12,30 +12,14 @@
 namespace spatial {
 
 class BenchmarkRunner {
-private:
+protected:
   std::string Key;
-  std::map<llvm::Instruction *, std::string> Result;
-  void evaluateUtil(llvm::CallInst *, int, int, int);
 
 public:
-  BenchmarkRunner() { Key = "ALIAS"; }
-  std::vector<llvm::Value *> extract(llvm::Instruction *Inst);
-  friend std::ostream &operator<<(std::ostream &OS, const BenchmarkRunner &B);
-  template <class Ty>
-  void evaluate(llvm::Instruction *Inst, std::set<Ty *>, std::set<Ty *>);
```

Comment:
Will the interface be incomplete without evaluate?

Reply:
Evaluate function has different signatures for both PTABenchmarkRunner and PointsToBenchmarkRunner that why we implemented evaluate function separately for both the classes.